tests/api/resource_test.py

```diff
@@ -223,9 +223,25 @@ def setup_resource(self):
     def test_render_GET(self):
         self.resource.get_data = MagicMock()
         result = self.resource.render_GET(REQUEST)
-        assert_call(self.resource.get_data, 0, False, False, True, True)
+        assert_call(self.resource.get_data, 0, False, False, True, True, None, None)
         assert "jobs" in result
 
+    def test_render_SEARCH(self):
+        request = build_request(query="test")
+        job1 = MagicMock()
```

Comment:
nit: we should probably set the `spec` on these to whatever the real type of these is so that the tests better match reality (e.g., so that the mocks fail should we ever remove `get_name` from the actual type, etc)